Src/PerformanceCollector/Shared/PerformanceCollectorModule.cs

```diff
@@ -125,6 +131,12 @@ public void Initialize(TelemetryConfiguration configuration)
                             throw new ArgumentNullException("configuration");
                         }
 
+                        if (!this.EnableIISExpressPerformanceCounters && IsRunningUnderIisExpress())
+                        {
+                            PerformanceCollectorEventSource.Log.RunningUnderIisExpress();
```

Comment:
Why would we write that at all? It is not an error, it is our by default behavior. We do not want customers to enable it back and get a deadlock. Let's report a verbose trace for diagnostics purposes only. If we want to write anything in debug output that should be "Performance counters collection is not available for issexpess" and that's it.